src/Core/src/Handlers/View/WindowsBatchPropertyMapper.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Maui.Handlers;
+
+#if WINDOWS
+class WindowsBatchPropertyMapper<TVirtualView, TViewHandler> : PropertyMapper<TVirtualView, TViewHandler>
+	where TVirtualView : IElement
+	where TViewHandler : IElementHandler
+{
+	// During mass property updates, this list of properties will be skipped
+	public static HashSet<string> SkipList = new(StringComparer.Ordinal)
+	{ 
+		// TranslationX does the work that is necessary to make other properties work.
+		nameof(IView.TranslationY),
+		nameof(IView.Scale),
+		nameof(IView.ScaleX),
+		nameof(IView.ScaleY),
+		nameof(IView.Rotation),
+		nameof(IView.RotationX),
+		nameof(IView.RotationY),
+		nameof(IView.AnchorX),
+		nameof(IView.AnchorY),
```

Comment:
Overall, this seems like it could be a temporary solution if it doesn't change any behavior.

Does this make ordering matter? What if you have code such as:

```csharp
view.TranslateX = 10;
view.TranslateY = 10;
// set the other properties *after* TranslateX
````

For this related change for Android:
* https://github.com/dotnet/maui/pull/3372

I believe it changed some behavior with handlers, so we put it in the _next_ .NET release. So this change might be more suitable for .NET 9, so it could get more testing?

Reply:
> I need something like up to 100ms to draw a complex grid

Have you thought about using SkiaSharp for this one piece of UI? You likely don't need all the rows & columns to be actual MAUI (& WinUI) Labels?